README.md

```diff
@@ -151,8 +153,13 @@ The output of the scan is parsed to generate the list of issues discovered by St
 ### Reporting
 
 _Reporting_ plugins output the issues found by the _tool_ plugins.
-The currently supported _reporting_ plugins are to print the output to a console and to write an _XML_ file that
-can be parsed by Jenkins.
+There are a number of existing _reporting_ plugins:
+- **Code Climate**: outputs a report that adheres to the [Code Climate Specification](https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#data-types)
+- **Jenkins**: writes an _XML_ file that can be parsed by [Jenkins](https://www.jenkins.io/)
+- **JSON**: writes a _JSON_ report to console or file
+- **Console**: prints the output to a console
+- **Do nothing**: a dummy plugin that outputs nothing. This can be useful when a reporting plugin is required but potential reporting side effects are undesirable
```

Comment:
This section does need to be updated since we have more than the original two reporting plugins. All of them are listed in a table in the README at https://github.com/sscpac/statick#reporting-plugins. I would prefer to leave the table to give details on reporting plugins. I'll make a suggestion on changing what we have in this section.